lenovo/legion/16irx9h/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  config,
+  ...
+}: {
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/gpu/nvidia/prime.nix
+    ../../../common/gpu/nvidia/ada-lovelace
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../../common/hidpi.nix
+  ];
+
+  boot.initrd.kernelModules = ["nvidia"];
```

Comment:
I don't think we should this option unless, we know why we are doing this:


```suggestion
```

https://github.com/NixOS/nixos-hardware/pull/1144

Reply:
OK. But it need to be fixed as well on [16irx8h](https://github.com/NixOS/nixos-hardware/blob/master/lenovo/legion/16irx8h/default.nix), am I right?